setup.py

```diff
@@ -16,8 +16,8 @@
     keywords='Firebase',
     packages=find_packages(exclude=['tests']),
     install_requires=[
-        'requests', 'sseclient', 'gcloud',
-        'oauth2client', 'requests_toolbelt', 'python_jwt',
-        'pycrypto'
+        'requests==2.11.1', 'sseclient', 'gcloud==0.17.0',
```

Comment:
Yeah that's right, the full source of sseclient comes bundled with Pyrebase now because at some point I have to make some changes to solve some streaming issues we are having.